tensorflow/python/ops/nn_ops.py

```diff
@@ -3493,6 +3493,37 @@ def leaky_relu(features, alpha=0.2, name=None):
     return gen_nn_ops.leaky_relu(features, alpha=alpha, name=name)
 
 
+@tf_export("nn.gelu")
+@dispatch.add_dispatch_support
+def gelu(features, approximate=True, name=None):
+  """Compute the Gaussian Error Linear Unit (GELU) activation function.
+
+  Args:
+    features: A `Tensor` representing preactivation values.
+    approximate: An optional `bool`. Defaults to `True`.
+      Whether to enable approximation.
+    name: A name for the operation (optional).
+
+  Returns:
+    A `Tensor` with the same type as `features`.
+
+  References:
+    [Gaussian Error Linear Units (GELUs)](https://arxiv.org/abs/1606.08415).
```

Comment:
Can we add doc tests with examples showing how to use this? https://www.tensorflow.org/community/contribute/docs_ref

Also, it would be helpful to add a brief description of GELUs in addition to the existing paper link.